test/mocking_classes.py

```diff
@@ -344,3 +348,60 @@ class DiscreteActionConvPolicy(DiscreteActionVecPolicy):
     def _get_in_obs(self, tensordict):
         obs = tensordict.get(*self.in_keys).diagonal(0, -1, -2).squeeze()
         return obs
+
+
+class DummyModelBasedEnv(ModelBasedEnv):
+    """
+    Dummy environnement for Model Based RL algorithms.
+    This class is meant to be used to test the model based environnement.
+    """
+
+    def __init__(
+        self,
+        device="cpu",
+        dtype=None,
+        batch_size=None,
+    ):
+        super(DummyModelBasedEnv, self).__init__(
+            WorldModelWrapper(
```

Comment:
Let's avoid building thing inside a caller, it's just syntax but it feels messy

Reply:
I'll changed that in the MBEnv PR instead